GridKit/Model/PhasorDynamics/Exciter/IEEET1/Ieeet1Impl.hpp

```diff
@@ -275,7 +275,7 @@ namespace GridKit
 
         ScalarT ind_low  = (this->sigmoid(Vrmin_ - x)) * (this->sigmoid(-f));
         ScalarT ind_high = (this->sigmoid(x - Vrmax_)) * (this->sigmoid(f));
-        return (1 - ind_low) * (1 - ind_high);
+        return (1.0 - ind_low) * (1.0 - ind_high);
       }
```

Comment:
We do need to define constants for GridKit. This should be something like
```suggestion
        return (ONE<RealT> - ind_low) * (ONE<RealT> - ind_high);
}
```
or

```suggestion
        auto one = ONE<RealT>;
        return (one - ind_low) * (one - ind_high);
}
```
Probably out of scope for this PR.

Reply:
Same holds for a few other places in the code.

Reply:
Happy to do it here, since we keep putting it off.

Reply:
Added a `Constants.hpp` file in [135e924](https://github.com/ORNL/GridKit/pull/305/commits/135e9242e7d59dc173a6545928e87117fc13ecae). Will need another PR to search and replace constants in the files that have not been touched by this PR.